qa/common/infer_util.py

```diff
@@ -90,37 +92,40 @@ def completion_callback(user_data, result, error):
 
 
 # Perform inference using an "addsum" type verification backend.
-def infer_exact(tester,
-                pf,
-                tensor_shape,
-                batch_size,
-                input_dtype,
-                output0_dtype,
-                output1_dtype,
-                output0_raw=True,
-                output1_raw=True,
-                model_version=None,
-                swap=False,
-                outputs=("OUTPUT0", "OUTPUT1"),
-                use_http=True,
-                use_grpc=True,
-                use_http_json_tensors=True,
-                skip_request_id_check=False,
-                use_streaming=True,
-                correlation_id=0,
-                shm_region_names=None,
-                precreated_shm_regions=None,
-                use_system_shared_memory=False,
-                use_cuda_shared_memory=False,
-                priority=0,
-                # 60 sec is the default value for L0_infer_valgrind
-                network_timeout=60.0):
+def infer_exact(
+        tester,
+        pf,
+        tensor_shape,
+        batch_size,
+        input_dtype,
+        output0_dtype,
+        output1_dtype,
+        output0_raw=True,
+        output1_raw=True,
+        model_version=None,
+        swap=False,
+        outputs=("OUTPUT0", "OUTPUT1"),
+        use_http=True,
+        use_grpc=True,
+        use_http_json_tensors=True,
+        skip_request_id_check=False,
+        use_streaming=True,
+        correlation_id=0,
+        shm_region_names=None,
+        precreated_shm_regions=None,
+        use_system_shared_memory=False,
+        use_cuda_shared_memory=False,
+        priority=0,
+        # 60 sec is the default value for L0_infer_valgrind
+        network_timeout=60.0,
+        kafka_obj=None):
```

Comment:
Mark future enhancement for test script that make "infer clients" (HTTP/GRPC/Kafka) to be some kind of callable